Package.swift

```diff
@@ -76,7 +76,10 @@ let includeDynamicLibrary: Bool = false
 let systemSQLitePkgConfig: String? = nil
 #else
 let includeDynamicLibrary: Bool = true
-let systemSQLitePkgConfig: String? = "sqlite3"
+var systemSQLitePkgConfig: String? = "sqlite3"
+if ProcessInfo.processInfo.environment["SWIFTCI_INSTALL_RPATH_OS"] == "android" {
```

Comment:
This is [only set by the official `build-script`](https://github.com/swiftlang/swift-package-manager/blob/cb829d83d016beb6607fde42136a079c37297d90/Utilities/bootstrap#L757), so it will only work in that scenario, which is fine as most builds of this repo happen that way.

Eventually, `systemLibrary` dependencies should be conditional on the platform, but that is not available yet.